cpp/src/parquet/arrow/arrow_schema_test.cc

```diff
@@ -906,22 +908,6 @@ TEST_F(TestConvertArrowSchema, ArrowFields) {
   // ASSERT_NO_FATAL_FAILURE();
 }
 
-TEST_F(TestConvertArrowSchema, ArrowNonconvertibleFields) {
-  struct FieldConstructionArguments {
-    std::string name;
-    std::shared_ptr<::arrow::DataType> datatype;
-  };
-
-  std::vector<FieldConstructionArguments> cases = {
-      {"float16", ::arrow::float16()},
```

Comment:
Instead of removing this test, can we use another Arrow datatype whose conversion to Parquet isn't supported?